### DISCUSSION TITLE
lost DLH icon on FIrefox menubar?

### DISCUSSION THREAD

**Discussion by williemyers123** [aclap-dev/video-downloadhelper#453] [Help and Q&A]
### Your question

I've re-installed DownLoad Helper on Firefox 115.10 for Mac and I no longer have the DownLoad Helper icon in  the toolbar? There's a space allocated for the icon and if I cursor over it, it says "VideoDownload Helper", but no icon to access the ext with?

### Addon & CoApp version and details

DownLoad Helper 8.2.2.8

**Comment by GrampaWildWilly** [comment]
You might want to take a look at this: #428.
